### PR TITLE
pos-print: deletion of registered device

### LINKED ISSUE #43: pos-devices: delete pos device mapping(configuration)
At this time there is no functionality for editing registered device, so if a device is moved to another office the old configurations probably should not be used any more, so the should be deleted

### PR DESCRIPTION
**Pull request by gadjevb**
Added logic that will enable the deletion by sourceIp of already registered cash register devices from the database.

Fixes #43

### CODE REVIEW

File: pos-print/src/main/java/com/clouway/pos/print/adapter/http/CashRegistersService.kt [hunk 1]
```diff
@@ -4,17 +4,18 @@ import com.clouway.pos.print.core.CashRegister
 import com.clouway.pos.print.core.ErrorResponse
```
Comment (mlesikov) [inline pos-print/src/main/java/com/clouway/pos/print/adapter/http/CashRegistersService.kt]: this service can be renamed
something like DeviceConfigurationService,DeviceService, DeviceBindingService, DeviceMappingsService 
will fits a lot better.

**Comment by gadjevb** [comment]
@mgenov made the required fixes, ready for review!

File: pos-print/src/test/java/com/clouway/pos/print/persistent/PersistentCashRegisterRepositoryTest.kt [hunk 2]
```diff
@@ -24,31 +24,49 @@ class PersistentCashRegisterRepositoryTest {
...
+    var expected = CashRegister("", "sourceIp", "any ip 1", "any description")
+    val id = repository.register(expected)
+    expected = CashRegister(id, "sourceIp", "any ip 1", "any description")
     val actual = repository.getBySourceIp("sourceIp").get()
 
     assert(actual.equals(expected))
   }
+
+  @Test
+  fun deleteBySourceIp() {
```
Comment (mgenov) [inline pos-print/src/test/java/com/clouway/pos/print/persistent/PersistentCashRegisterRepositoryTest.kt]: It's deleted by id now, not by source ip. 

deleteCashRegister
deleteNonExistingCashRegister

**Comment by gadjevb** [comment]
@mgenov made the required changes, ready for review!

**Pull request merged**